src/core/map_printer.h

```diff
@@ -165,6 +165,10 @@ class MapPrinterOptions
 	 *  Only available in MapPrinterOptions::Raster mode.
 	 */
 	bool simulate_overprinting;
+
+    /** Controls whether to save a world file when outputting an image
+     */
+    bool save_world_file;
```

Comment:
`save_world_file` member should be initialized in class constructor. Otherwise UBSAN complains:
```
/home/lpechacek/oomapper/src/core/map_printer.cpp:720:14: runtime error: load of value 116, which is not a valid value for type 'bool'
```

Reply:
Fixed thanks.